src/Manager.js

```diff
@@ -442,15 +449,14 @@ class GiveawaysManager extends EventEmitter {
             channel.messages.cache.get(packet.d.message_id) ||
             (await channel.messages.fetch(packet.d.message_id));
         if (!message) return;
-        const reaction = message.reactions.cache.get(
-            giveaway.reaction || this.options.default.reaction
-        );
+        const reaction = message.reactions.cache.get(giveaway.reaction);
         if (!reaction) return;
         if (reaction.emoji.name !== packet.d.emoji.name) return;
         if (reaction.emoji.id && reaction.emoji.id !== packet.d.emoji.id) return;
         if (packet.t === 'MESSAGE_REACTION_ADD') {
             if (giveaway.ended) return this.emit('endedGiveawayReactionAdded', giveaway, member, reaction);
             this.emit('giveawayReactionAdded', giveaway, member, reaction);
+            if (giveaway.options.isDrop == true && reaction.count >= giveaway.winnerCount) this.end(giveaway.messageID).catch(() => {});
```

Comment:
```suggestion
            if (giveaway.options.isDrop && reaction.count >= giveaway.winnerCount) this.end(giveaway.messageID).catch(() => {});
```
Not needed because you already added that it makes sure that it only is a boolean

Reply:
Yep thanks for the correction.